### DISCUSSION TITLE
how did you used list-view builder

### DISCUSSION THREAD

**Discussion by wetcats** [abhi3940/e_vahak#13] [Q&A] [answered]
how did you use your list view builder to only show the tickets booked by the user I want to do the same thing for a project of mine it would be a great help if you could answer this

**Comment by abhi3940** [comment, marked as answer]
we have used streamproviders in our application which returns list of tickets the user has booked with the help of firebase query 
but if you don't want to use streamprovider you can use simple ternary oprator
